peas/sensors.py

```diff
@@ -141,20 +150,20 @@ def capture(self, use_mongo=True, send_message=True):
                 if send_message:
                     self.send_message({'data': data}, channel='environment')
             except yaml.parser.ParserError:
```

Comment:
I think I didn't know this when I wrote this a long time ago, but you can pass a tuple for except:

```
except (yaml.parser.ParserError, ValueError, TypeError, Exception) as e:
```

However if we have `Exception` at the end and we are not doing anything different for the different exceptions then really we just need to capture the generic.

Reply:
Hmm, I didn't know that either, though as you say it doesn't do much to include both a class and super-class in the the same tuple.